app/controllers/instruments_dashboard_controller.rb

```diff
@@ -0,0 +1,35 @@
+class InstrumentsDashboardController < ApplicationController
```

Comment:
I could be convinced that `ReservationsDashboardController` or something like that is a better name. I originally used this name because I was looping over the all the instruments and finding the active one, but since they want to only show active reservations, it became easier to loop over reservations. I'd also be open to other names.

Reply:
I have no real opinion here, personally I think `InstrumentsDashboardController` makes sense.